### ISSUE TITLE
Suggestion: Add a 'Bench' for Alternate Monsters to Team View

### ISSUE THREAD

**Issue by adangtran87** [madcowfred/padherder-issues#620]
Currently, when you have more than 5 monsters that would be viable on a team, you have to make multiple teams to show the possibilities. My suggestion would be adding a 'bench'/alternate list, where you can drag monsters in and out of the team (dynamically updating team stats) to basically explore your options with a particular leader. 

I imagine the experimentation would not need to be stored in the database. A user would create a 'base' team that would normally be loaded when viewing that particular team (ie. user reassembles team for a particular dungeon and leaves page. Returning to the page would show the base team). The description field could then be used to explain the use cases for each alternate (aka replace X with Perseus in skill bind dungeons).

**Comment by FaxCelestis** [comment]
I would use this so much.

**Comment by wzhang125** [comment]
Yeah, this is a great idea. I have a lot of clutter in my teams page due to the many variations I need for various dungeons - this would tidy things up in a flash. This feature would also make teambuilding for new dungeons (and new leaders) a lot more streamlined.
